### ISSUE TITLE
Defenders of Darrowshire

### ISSUE THREAD

**Issue by Djfe** [AeroScripts/QuestieDev#1435]
## Description
There are more type of ghouls that allow the completion of this quest, than Carlin mentions.
Gibbering Ghouls for example.

## Wowhead link
https://classic.wowhead.com/quest=5211/defenders-of-darrowshire#english-comments
https://classic.wowhead.com/npc=8531/gibbering-ghoul#comments

## Questie version
5.0.1

**Comment by Djfe** [comment]
And there are no sword icon at Terrordale and Plaguewood where some mobs are as well.

**Comment by BreakBB** [comment]
@Djfe With the latest version we're changed our logic to draw icons. We're prioritising the ones close to the starting NPC. That could be the reason why there're no icons at Terrordale and Plaguewood.

Will look into the other NPCs

**Comment by Djfe** [comment]
won't that lead to people overfarming those spots?
is it really an issue? is the map overcluttered with icons?

will there be a setting for this? (opt-out or opt-in)

**Comment by BreakBB** [comment]
It totally depends on the objective. There're some NPCs in the DB which have **a lot** of spawns and drawing each of them could (at this point) drastically reduce the performance for some users.

We discussed having this as an option. Since there is more advanced logic behind it (not just "draw X icons") we decided to not have it totally as an option, but afaik there is a slider belonging to this in the advanced tab.
